### DISCUSSION TITLE
OnRemoteConnectionState & overall network efficiency

### DISCUSSION THREAD

**Discussion by robvld** [FirstGearGames/FishNet#547] [Feature Request]
By default Share Ids's is enabled. Would i not have checked OnRemoteConnectionState on the client i would not have known the server would send data to each connected client when other clients are connecting/disconnecting.

Would it not be better, imho, for the network library to send as little information by default and have users enable it when they need it?

thanks.

**Comment by FirstGearGames** [comment]
The bandwidth used by sharing Ids is incredibly insignificant and there is absolutely no security risk involved as no sensitive information is sent. 

ShareIds simply shares who owns objects/predicted spawned objects, and minor things like that. 

Most developers prefer the setting on so that's why it's the default.
